app/internal/hebrew_date_view.py

```diff
@@ -0,0 +1,70 @@
+from hebrew_numbers import int_to_gematria
+from pyluach import dates, hebrewcal
+
+
+def get_month_name_by_num(date) -> str:
+    """Returns the Hebrew name date for the specific day.
+
+        Args:
+            date: The requested date.
+
+        Returns:
+            A Hebrew name date.
+        """
+    hebrew_dates_translate_dict = {
```

Comment:
1. Define this outside, we usually don't want to define constants inside functions.
2. Constants should be `UPPER_CASE_WITH_UNDERSCORES`
3. Prefer not using the type of the structure in the name ([Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation#:~:text=Hungarian%20notation%20is%20an%20identifier,in%20some%20dialects%20its%20type.)): remove `_dict`.
4. Probably can choose a more descriptive name: `EN_TO_HEB_MONTH_NAMES`
5. You can use `gettext()` instead, and leave the translation to the translators.

Reply:
i prefer not use gettext .my way is easierto me in leap year case